app/code/Magento/Review/Api/ReviewRepositoryInterface.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Review\Api;
+
+/**
+ * @api
+ */
+interface ReviewRepositoryInterface
```

Comment:
Please split this repository into individual interfaces like it is done in MSI https://github.com/magento-engcom/msi/tree/2.3-develop/app/code/Magento/InventoryApi/Api

Skip repository for now.